### PR TITLE
Re styling

### PR DESCRIPTION
**Pull request by DannaTB**
Slight adjustments to fix discovered bug

### CODE REVIEW

File: components/qualtrics/injectionLoader.mjs [hunk 1]
```diff
@@ -7,7 +7,7 @@ export async function injectionLoader(graphObjStr) {
   
   // load (this) module
   // ! TODO change import location to read from somewhere
-  let modulePromise = import("https://cdn.jsdelivr.net/gh/2022-CITS3200-GraphTeam/CITS3200-GroupProject@v0.2.1/components/qualtrics/injection.min.mjs");
+  let modulePromise = import("http://localhost:5500/components/qualtrics/injection.mjs");
```
Comment (met4000) [inline components/qualtrics/injectionLoader.mjs:10]: (See comment on the other link)

File: components/qualtrics/consts.mjs [hunk 2]
```diff
@@ -1,2 +1,2 @@
 // ! TODO change import location to read from somewhere
-export const BASE_URL = "https://cdn.jsdelivr.net/gh/2022-CITS3200-GraphTeam/CITS3200-GroupProject@v0.2.1";
\ No newline at end of file
+export const BASE_URL = "http://localhost:5500";
```
[the remark this answers is not in the document]
Comment (DannaTB) [reply components/qualtrics/consts.mjs]: Links updated

File: components/qualtrics/injectionLoader.mjs (hunk 1 above)
Comment (DannaTB) [reply components/qualtrics/injectionLoader.mjs]: Link updated

**Comment by DannaTB** [comment]
Code conflicts resolved

**Review by met4000 — APPROVED** [review]

**Pull request merged**